public/v4/apps/typesense.yml

```diff
@@ -0,0 +1,44 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: typesense/typesense:$$cap_version
+        restart: on-failure
+        ports:
+            - '8108:8108'
```

Comment:
I assume this is an HTTP port. In that case use `containerHttpPort` - there are many examples of it in the repo.